client/interview-controller.js

```diff
@@ -0,0 +1,79 @@
+// Description:
+//   A Hubot script which helps hrs to control interviews.
+//
+// Commands:
+//  begin group Huntflow Reloaded
+//    begin admin
+//      hubot удалить интервью - remove an interview from the list
+//    end admin
+//  end group
+//
```

Comment:
For this configuration the hubot-help script is not ready. So it fails with an error.

Reply:
The PR to the hubot-help script is ready so you can keep it here. 
And let's rephrase it to `removes the non-expired interview of the specified candidate`